bootstrap.ps1

```diff
@@ -0,0 +1,20 @@
+$Desktop = [Environment]::GetFolderPath("Desktop")
```

Comment:
This script can be removed, because it has been integrated into the [`configuration`](https://github.com/StefanGreve/configuration/) repository.